README.md

```diff
@@ -1,5 +1,9 @@
 ﻿# Z-Wave Home Automation library for .NET
 
+## About this fork
+I've forked this to remove NLog as the project I am using this library in doesn't use NLog thus adding more libraries seem useless.
+This fork also contains WIP's for adding new commandclasses to the library when I stumble upon one that's missing.
+
```

Comment:
Agreed, the removal of NLog is more a thing that I use in my own projects as it has a different way of logging.
Furthermore this is basically my first pull request ever so I don't know if you can ignore the other changes beside the ones regarding the CentralScene ones?